hub/management/commands/import_socio_economic_status_data.py

```diff
@@ -0,0 +1,128 @@
+from django.core.management.base import BaseCommand
+
+import pandas as pd
+from tqdm import tqdm
+
+from hub.models import Area, AreaData, DataSet, DataType
+
+
+class Command(BaseCommand):
+    help = "Import data about socio-economic statuses"
+
+    source_url = "https://commonslibrary.parliament.uk/find-the-socio-economic-status-of-people-living-in-england-and-wales-by-constituency/"
+    data_url = "https://data.parliament.uk/resources/constituencystatistics/PowerBIData/Census%202021/NS-SEC_2021.xlsx"
+
+    message = "Importing constituency socio-economic data"
+    defaults = {
+        "label": "Socio-Economic Status",
+        "description": "Socio-economic status as of the 2021 census",
+        "data_type": "percent",
+        "category": "place",
+        "source_label": "House of Commons Library",
+        "is_range": True,
+        "source": source_url,
+        "source_type": "xlxs",
+        "data_url": data_url,
+        "table": "areadata",
+        "exclude_countries": ["Scotland", "Northern Ireland"],
+        "comparators": DataSet.numerical_comparators(),
+        "default_value": 10,
+    }
+
+    label_names = {
```

Comment:
Might be worth a comment here to explain these are because of character length limits.

Or we could avoid this and just update the table to increase the label length. There's no reason it's set to 200 other than I arbitrarily picked that as probably long enough.

Reply:
wouldn't solve the display issues though which is a secondary thing.

Reply:
Label changes now has the display looking like this:
![image](https://github.com/mysociety/local-intelligence-hub/assets/65429064/885eef4b-bce9-4870-a3b8-414b27e34c57)
Which obviously looks better, but I'm not sure if the label-wrangling is okay from a data integrity point of view?

Reply:
As a table, with no more label mangling:
![image](https://github.com/mysociety/local-intelligence-hub/assets/65429064/f6c354a1-b167-43f4-947a-b23227c70318)